arch/Kconfig

```diff
@@ -69,6 +69,16 @@ config USERSPACE
 	This feature is under heavy development and APIs related to it are
 	subject to change, even if declared non-private.
 
+config PRIVILEGED_STACK_SIZE
+	int "Size of privileged stack"
+	default 256
```

Comment:
It's a guess.  We can change the defaults if we run into issues.  Aside from network calls, i have to wonder how deep some of these call chains are.

Reply:
wrt network stack I think we can cross that bridge when we get there, as there is currently no network functionality exposed via system calls.
if 256 is a wild guess, let's make sure we have stack guards turned on when we test, so that we know when we have overflowed.